modules/react/button/lib/TertiaryButton.tsx

```diff
@@ -153,14 +152,15 @@ export const TertiaryButton = createComponent('button')({
     const allContainerStyles = allCaps
       ? {
           ...containerStyles,
-          ...type.variant.caps,
```

Comment:
This is why `caps` also adding a `fontWeight` is problematic. Glad it's gone. ✌🏻